runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -147,8 +147,20 @@ static int getCoresPerLocale(int nomultithread, int32_t localesPerNode) {
   }
 
   memset(buf, 0, buflen);
-  if (chpl_run_utility1K("sinfo", argv, buf, buflen) <= 0)
+  if (chpl_run_utility1K("sinfo", argv, buf, buflen) <= 0) {
     chpl_error("Error trying to determine number of cores per node", 0, 0);
```

Comment:
Won't we hit this fatal chpl_error still before trying the fallback?

Reply:
It's not fatal, `sinfo` prints the message but exits with a zero status.